src/routes/prof-nidurstodur/KosningaProf.js

```diff
@@ -177,9 +177,12 @@ class Kosningaprof extends PureComponent {
       if (isImportant(currentValue)) {
         const newValue = _cleanAnswer();
         this.onChange(id)(newValue);
+      } else if(currentValue) {
+         this.onChange(id)(`${currentValue}!`);
       } else {
```

Comment:
I think this bug was fixed in https://github.com/kristjanmik/kjosturett-web/commit/f9a88368e13159dbc05694c865d77059d348634b if I understood your comment correctly